### DISCUSSION TITLE
Converting Geometries to VectorTiles and Saving in PBF Format using NTS?

### DISCUSSION THREAD

**Discussion by miganmht** [NetTopologySuite/NetTopologySuite#728] [Q&A] [answered]
Hello NTS Community,

I am currently working with NetTopologySuite  and exploring the possibility of converting geometries into VectorTiles, followed by saving them in the Protocol Buffers (PBF) format. Specifically, I am seeking guidance on the step-by-step process and best practices for achieving this.

If anyone has experience or insights into using NTS for creating VectorTiles from geometries and saving them in PBF, I would greatly appreciate any assistance or pointers you could provide.

**Comment by FObermaier** [comment, marked as answer]
Please have a look at the NetTopologySuite.IO.VectorTiles project/package.
